google-cloud-video-stitcher/samples/acceptance/helper.rb

```diff
@@ -35,10 +37,28 @@ class StitcherSnippetSpec < Minitest::Spec
   let(:slate_uri) { "https://storage.googleapis.com/cloud-samples-data/media/ForBiggerEscapes.mp4" }
   let(:updated_slate_uri) { "https://storage.googleapis.com/cloud-samples-data/media/ForBiggerJoyrides.mp4" }
 
+  let(:gcdn_cdn_key_id) { "my-gcdn-test-#{(Time.now.to_f * 1000).to_i}" }
+  let(:gcdn_cdn_key_name) { "projects/#{project_id}/locations/#{location_id}/cdnKeys/#{gcdn_cdn_key_id}" }
+  let(:akamai_cdn_key_id) { "my-akamai-test-#{(Time.now.to_f * 1000).to_i}" }
+  let(:akamai_cdn_key_name) { "projects/#{project_id}/locations/#{location_id}/cdnKeys/#{akamai_cdn_key_id}" }
+
+  let(:hostname) { "cdn.example.com" }
+  let(:updated_hostname) { "updated.example.com" }
+  let(:gcdn_key_name) { "gcdn-key" }
+  let(:updated_gcdn_key_name) { "updated-gcdn-key" }
+  let(:gcdn_private_key) { "VGhpcyBpcyBhIHRlc3Qgc3RyaW5nLg==" }
```

Comment:
Is it possible to generate these private keys ?

Reply:
Possibly, but I don't think this is needed here. The private key here is literally "This is a test string." The updated private key is "This is an updated test string."